### DISCUSSION TITLE
Bypasses for school internet filtering? (Content Keeper)

### DISCUSSION THREAD

**Discussion by crashy102** [3kh0/ext-remover#1171] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE]
My school swapped from the lightspeed extension to Content Keeper, which is an internet filtering thingy that also has an extension. Before my school forced all of the computers to 120 but after they switched to CK, Skiovox stopped working because of web filtering, and when I used it at home, the filtering was gone completely. Does anyone know how to bypass this?

**Comment by funkymokey** [comment]
Now you can dwell in the pool of pain with me. There isn't a reliable way to keep the filter off. Something cool I found out on streaming services is using your hotspot to load the website, then connecting back to the school wifi will leave the filter on, but that website and the contents will stay if loaded and continue working with the filtered internet
(edit for the third time don't forget to turn off the extension)

**Reply by HelloIloveGimkit2** [reply]
you could just use rigtools.

**Reply by HelloIloveGimkit2** [reply]
here's the steps


go to this url:
devtools://devtools/bundled/devtools_app.html

then this one:
devtools://devtools/bundled/devtools_app.html?experiments=true&ws=fprkr6-8080.csb.app
double-click on the blank request in the network tab
then
get the extension ID and copy it then go back to the newly created rigtools dashboard and click "extension ID" then paste your extension's ID and there you can disable it.

When you are on the filesystem:chrome-extension://ext-id/temporary/index.html
you may be able to disable other extensions

**Reply by stuartcat** [reply]
sadly this will only disable the extension content keeper is a much more powerful blocking system that is more network level 
personally mines certificates have expired so it no longer works

**Reply by HelloIloveGimkit2** [reply]
> sadly this will only disable the extension content keeper is a much more powerful blocking system that is more network level personally mines certificates have expired so it no longer works

I legit said that you could use it for the extension read all the replies before making a f3cking response

**Reply by sadgimkitgamer** [reply]
![image](https://github.com/user-attachments/assets/58c0f422-8bf9-4bcd-9a14-f647fa6a5a64)

I get this image when I try to use the thing. You know what went wrong?

**Reply by HelloIloveGimkit2** [reply]
if you get that error try a different one as that websocket/link is down

**Reply by HelloIloveGimkit2** [reply]
try the edited link

**Reply by sadgimkitgamer** [reply]
![Screenshot 2024-10-02 6 42 15 PM](https://github.com/user-attachments/assets/58e360fa-12d6-4c93-905a-edb0b2dafd61)
now I get this error

**Reply by HelloIloveGimkit2** [reply]
check your version you may be on v129 or above which is patched

**Reply by sadgimkitgamer** [reply]
NOOOOO

IM ON v129! 

do u have any other sugguestions?

**Reply by HelloIloveGimkit2** [reply]
nope idk

**Reply by sadgimkitgamer** [reply]
do u have anything else now?

**Reply by thatboyo** [reply]
dude same, what school district you at? crashy102?

**Reply by savirduggal-sketch** [reply]
how to bypass contentkeeper

**Reply by nathblade16** [reply]
> how to bypass contentkeeper

to be honest these kinds of questions asking just "how to bypass contentkeeper" is what a skid would ask

**Reply by aidenshamberger49-alt** [reply]
> > how to bypass contentkeeper
> 
> to be honest these kinds of questions asking just "how to bypass contentkeeper" is what a skid would ask
bro what is a skid

**Reply by nathblade16** [reply]
> > > how to bypass contentkeeper
> > 
> > to be honest these kinds of questions asking just "how to bypass contentkeeper" is what a skid would ask
> bro what is a skid

Script Kidde, the kind of person who just follows instructions not knowing how they work or if it will work or not

**Reply by notpulsewave** [reply]
i have ck accept i kinda turned off all filtering so everything in online but ck is offline
<img width="255" height="69" alt="Screenshot 2026-07-25 8 41 57 PM" src="https://github.com/user-attachments/assets/7fb579c7-9db9-4c02-90ab-44ad34a85505" />

**Reply by Darkn-star** [reply]
Where are you supposed to go though???

**Comment by HelloIloveGimkit2** [comment]
Also just use a VPN if those are blocked then rigtools for the extension if you wanna do the internet filtering just use something else for dns even your phone could work

if none of those work (rig tools should you're an idiot if you can't use it): cry in a corner
elseif 1 did work then
profit

**Reply by funkymokey** [reply]
RigTools works, but that wasn't the point. It's about network filtering. What DNS and VPN are you referring to that works?

**Reply by HelloIloveGimkit2** [reply]
bro are you slow? there's a ton of VPNs there's a kajig for proton VPN to use it

**Reply by HelloIloveGimkit2** [reply]
> bro are you slow? there's a ton of VPNs there's a kajig for proton VPN to use it

you prob don't know what a kajig is, it's an exploit, webview, VPN, proxy almost every exploit packed into a repo

**Reply by funkymokey** [reply]
thanks for bringing up the kajig, I was able to use proton vpn to make my own little vpn

**Reply by thatboyo** [reply]
is the instructions for proton vpn on tn website?

**Comment by Perplexedykwhat** [comment]
maybe try ExtHang3r? idk tbh

**Comment by nathblade16** [comment]
ck-p0ision [link](https://contentgiver.github.io/guides/ck-p0ision)

**Reply by ghost** [reply]
Is this patched? I'm on chrome 130 and it doesn't seem to work for me.

**Reply by nathblade16** [reply]
> Is this patched? I'm on chrome 130 and it doesn't seem to work for me.

its not patched but might not work for some people

**Comment by S-PScripts** [comment]
https://github.com/3kh0/ext-remover/discussions/1532

**Comment by S-PScripts** [comment]
no
